tests/ui/issues/issue-31769.stderr

```diff
@@ -1,8 +1,14 @@
-error[E0518]: attribute should be applied to function or closure
+error: `#[inline]` attribute cannot be used on structs
   --> $DIR/issue-31769.rs:2:5
    |
 LL |     #[inline] struct Foo;
-   |     ^^^^^^^^^ ----------- not a function or closure
+   |     ^^^^^^^^^
+   |
+help: `#[inline]` can be applied to functions
+  --> $DIR/issue-31769.rs:2:5
+   |
+LL |     #[inline] struct Foo;
+   |     ^^^^^^^^^
```

Comment:
Hmmm I've just noticed this is super ugly with the `help` getting its own codespan, let me see if I can fix that